.travis/custom_release.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+set -e
+set -x
+
+NODE_ENV=production npm run build
+
+if [ "${TRAVIS_BRANCH}" = "master" ]
+then
+    for env in ci qa
+    do
+        echo "PUSHING ${env}-beta"
+        rm -rf ./dist/.git
+        .travis/release.sh "${env}-beta"
+    done
+fi
+
+if [ "${TRAVIS_BRANCH}" = "master-stable" ]
```

Comment:
Does this need to change to match `sources-ui` ( `stable` ) ?

Reply:
Updated